tests/utils/nitpick.js

```diff
@@ -505,3 +566,83 @@ module.exports.saveResults = function(passed, resultsObject) {
 		++textIndex;
     }
 }
+
+module.exports.verifyClientProfile = function() {
+    for (var i = 0; i < arguments.length; ++i) {
+        if (arguments[i] == "Any") {
```

Comment:
I still thinks this needs improvement.  I think the way it should work is it accepts an optional argument, which, if defined, we parse to see if it contains specifications for the OS, the device, or high/low end.

Reply:
It does accept arguments - any number you want.   For a Mac specific test you would call `nitpick.verifyClientProfile("Desktop-Mac").

Reply:
No, I mean, it should accept an argument like "-clientProfile=<profile>" where <profile> optionally contains OS, device, and/or spec information, instead of hard-coding each case.

Reply:
How about we pass an object?

Say we want to run on any desktop:
clientProfile = { display: "Desktop" }

To test Rift on Windows:
clientProfile = { display: "Rift", operatingSystem: "Windows" }

Missing properties imply that all values are valid.
And if you want the test to run on either of these 2, then you pass 2 objects.

Reply:
that sounds good :+1: